pkg/executor/executor.go

```diff
@@ -163,19 +165,28 @@ func DoBuild(dockerfilePath, srcContext, destination, snapshotMode string, docke
 		return err
 	}
 
-	wo := remote.WriteOptions{}
-	if ref != nil {
-		wo.MountPaths = []name.Repository{ref.Context()}
-	}
-	pushAuth, err := authn.DefaultKeychain.Resolve(destRef.Context().Registry)
+	destRef, err := name.NewTag(destination, name.WeakValidation)
 	if err != nil {
 		return err
 	}
+
 	sourceImage, err = mutate.Config(sourceImage, imageConfig.Config)
 	if err != nil {
 		return err
 	}
 
+	if opts.TarPath != "" {
+		return tarball.Write(opts.TarPath, destRef, sourceImage, nil)
```

Comment:
This was changed to `WriteToFile` in a more recent commit, could you update the dependencies and this line?

Reply:
Good catch. Just merged in another PR that updated the dep, and redid this.